### PR TITLE
Fix parsing crashes

### PR DESCRIPTION
**Pull request by Isakdl**
- Will ignore empty rows without crashing,
- Will ignore rows with empty key value
- Fix crash if a translation value is missing for a language, now defaults to an empty string
- Print a warning to the console if an empty string is found when parsing

### CODE REVIEW

File: lib/src/parser/translation_parser.dart [hunk 1]
```diff
@@ -33,12 +33,25 @@ class TranslationParser {
...
+        var itemValue;
+        //incase value does not exist
+        if(index < item.values.length) {
+          itemValue = item.values[index];
+        } else {
+          itemValue = '';
+        }
+
+        if(itemValue == '') {
+          print('WARNING: empty string in lang: '+ document.languages[index] + ', key: '+ item.key);
```
Comment (gocal) [inline lib/src/parser/translation_parser.dart]: please use logX(...) method e.g. logd(

File: lib/src/parser/translation_parser.dart [hunk 2]
```diff
@@ -33,12 +33,25 @@ class TranslationParser {
...
+        }
+        
         final itemPlaceholders = _findPlaceholders(itemValue);
 
         final builder = builders[index];
         final parser = parsers[index];
 
+
```
Comment (gocal) [inline lib/src/parser/translation_parser.dart]: redundant space

**Review by gocal — CHANGES_REQUESTED** [review]

File: lib/src/parser/translation_parser.dart (hunk 1 above)
Comment (Isakdl) [reply lib/src/parser/translation_parser.dart]: changed to Log.i(...) from util package

File: lib/src/parser/translation_parser.dart (hunk 2 above)
Comment (Isakdl) [reply lib/src/parser/translation_parser.dart]: removed

**Pull request merged**